packages/xrpc-server/tests/bodies.test.ts

```diff
@@ -237,4 +246,31 @@ describe('Bodies', () => {
 
     await expect(promise).rejects.toThrow('request entity too large')
   })
+
+  it('requires any parsable Content-Type for blob uploads', async () => {
+    // not a real mimetype, but correct syntax
+    await client.call('io.example.blobTest', {}, randomBytes(BLOB_LIMIT), {
+      encoding: 'some/thing',
+    })
+  })
+
+  // @TODO: figure out why this is failing dependent on the prev test being run
+  // https://github.com/bluesky-social/atproto/pull/550/files#r1106400413
+  it.skip('errors on an empty Content-type on blob upload', async () => {
```

Comment:
Tt fails because of the following line that break express' control flow.

https://github.com/bluesky-social/atproto/blob/169486a3fef64da086d1499ba9ba743fdfef0af2/packages/xrpc-server/src/server.ts#L194-L198

Indeed, when an error is triggered in the body (e.g. because of the max-size), the error is now propagated in two places: `next` (for the `error` handler) as well as the place where the readable `body` was being consumed (which may, or may not, write to the response, call next, etc.)